h/accounts/views.py

```diff
@@ -38,6 +38,70 @@ def ajax_payload(request, data):
     return payload
 
 
+def to_xhr_response(request, non_xhr_result, form):
```

Comment:
I think the next step will be to get the new group form to use this, so that'll involve moving it to a more generic place and if necessary making the code generic, but I'd like to leave that for a later pr.